cypress/tests/overview/settings-cluster-general.cy.ts

```diff
@@ -0,0 +1,96 @@
+import { tab } from '../../views/tab';
+
+describe('Test Cluster General settings', () => {
+  before(() => {
+    cy.visit('');
+    cy.wait(15000);
+    cy.switchToVirt();
+    cy.visitVirtPerspectiveOverview();
+    tab.navigateToSettings();
+  });
+
+  describe('Check installed version appears', () => {
+    it('check version', () => {
+      cy.get('[data-test="detail-item-value"]').contains('4');
+      cy.get('[data-test="detail-item-value"]').contains('Up to date');
+    });
+  });
+
+  describe('Test Live migration', () => {
+    it('set live migration limits', () => {
+      cy.contains('General settings').click();
+      cy.contains('Live migration').click();
+      cy.get('input[name="parallelMigrationsPerCluster"]').clear().type('4');
+      cy.get('input[name="parallelOutboundMigrationsPerNode"]').clear().type('1');
+      cy.contains('Set live migration network').click();
+      cy.wait(1000);
+    });
+
+    it('verify live migration limits in HCO', () => {
+      const maxPerCluster = '.spec.liveMigrationConfig.parallelMigrationsPerCluster';
+      const maxPerNode = '.spec.liveMigrationConfig.parallelOutboundMigrationsPerNode';
+      const spec = '.spec.liveMigrationConfig';
+      const cmd = `oc get -n openshift-cnv hyperconverged kubevirt-hyperconverged -o jsonpath='{${spec}}'`;
+      cy.exec(cmd).then((res) => {
+        cy.task('log', res.stdout);
+      });
+      cy.checkHCOSpec(maxPerCluster, '4', true);
+      cy.checkHCOSpec(maxPerNode, '1', true);
+    });
+  });
+
+  describe('Test Memory density', () => {
+    it('ID(CNV-11225) enable memory density', () => {
+      cy.contains('Memory density').click();
+      cy.get('[id="memory-density-feature"]').find('input[type="checkbox"]').check({ force: true });
+    });
+
+    it('verify higherWorkloadDensity in HCO', () => {
+      const percentage = '.spec.higherWorkloadDensity.memoryOvercommitPercentage';
+      cy.checkHCOSpec(percentage, '150', true);
+      cy.wait(10000);
+    });
+
+    it('ID(CNV-11225) disable memory density', () => {
+      cy.get('[id="memory-density-feature"]')
+        .find('input[type="checkbox"]')
+        .uncheck({ force: true });
+    });
+
+    it('verify higherWorkloadDensity in HCO', () => {
+      const percentage = '.spec.higherWorkloadDensity.memoryOvercommitPercentage';
+      cy.checkHCOSpec(percentage, '100', true);
+    });
+  });
+
+  describe('Test Automatic images download', () => {
+    it('ID(CNV-11223) disable centos-stream-9-image-cron', () => {
+      cy.contains('Automatic images download').click();
+      cy.get('#centos-stream9-image-cron-auto-image-download-switch')
+        .find('input.pf-v5-c-switch__input')
+        .uncheck({ force: true });
+      cy.wait(10000);
+    });
+
+    it('delete centos-stream9 volumesnapshot', () => {
+      const cmd =
```

Comment:
we have no assertions here (and below) that action was successful

Reply:
cypress will report failure if the command is failed to execute.